research/src/sources/fast.json5

```diff
@@ -105,6 +105,12 @@
     {
       "name": "Tree view",
       "url": "https://explore.fast.design/components/fast-tree-view"
+    },
+    {
+      "name": "Data Grid",
+      "openUIName": "Flex",
+      "definition": "The data grid component enables authors to display an array of data in a tabular layout. The layout can include a header region where a title can be displayed for each column.",
+      "url": "https://explore.fast.design/components/fast-data-grid"
```

Comment:
This seems out of place - I think the intent of this control is fundamentally different than the other presentational ones listed here.

Reply:
You're right, I guess my reasoning here was to gather all layout capabilities of the different design systems and I ended up misreading the purpose of Data Grid. I agree that this is more of a table layout than any other thing. Will make the changes.